src/main/java/school/mjc/stage0/conditions/task1/IntegerComparison.java

```diff
@@ -4,8 +4,8 @@ public class IntegerComparison {
     public static void main(String[] args) {
         int x = 50;
         int y = 50;
-         (x  y) {
-            System.out.println("Hello World!");
+        _ (x _ y) {
```

Comment:
maybe change to double underscore as in the readme or change the readme to include single symbol

Reply:
fixed